diabetes_regression/scoring/score.py

```diff
@@ -36,16 +37,12 @@ def init():
     # load the model from file into a global object
     global model
 
-    # AZUREML_MODEL_DIR is an environment variable created during service
-    # deployment. It contains the path to the folder containing the model.
-    path = os.environ['AZUREML_MODEL_DIR']
-    model_path = None
-    for root, dirs, files in os.walk(path):
-        for file in files:
-            if '.pkl' in file:
-                model_path = os.path.join(path, file)
-    if model_path is None:
-        raise ValueError(".pkl model not found")
+    # we assume that we have just one model
+    # AZUREML_MODEL_DIR is an environment variable created during deployment.
+    # It is the path to the model folder
+    # (./azureml-models/$MODEL_NAME/$VERSION)
+    model_path=Model.get_model_path(os.getenv("AZUREML_MODEL_DIR").split('/')[1])
```

Comment:
```suggestion
    model_path=Model.get_model_path(os.getenv("AZUREML_MODEL_DIR").split('/')[-2])
```

I guess this is more robust, since the beginning of the path might change in a future AML SDK.